pkgs/top-level/python-packages.nix

```diff
@@ -5973,16 +5973,15 @@ in modules // {
     };
   };
 
-  i3pystatus = buildPythonPackage rec {
+  i3pystatus = buildPythonApplication rec {
     name = "${pname}-${version}";
```

Comment:
Good catch. Being a collection of scripts `buildPythonApplication` fits better than `buildPythonPackage`.

That actually also means that this expression needs to be moved elsewhere, somewhere in `pkgs/applications`, because `python-packages.nix` is meant only for packages, not applications.